packages/shared/src/lib/links.ts

```diff
@@ -24,3 +24,9 @@ export const isPreviewDeployment = (() => {
 
   return !!value;
 })();
+
+export const links = {
+  notifications: {
+    firstNotification: 'https://r.daily.dev/notifications',
+  },
+};
```

Comment:
We already keep track of these in `lib/constants.ts` please keep all links there.

Reply:
The constants file was actually looking a bit messy with all the links around and other constants. So I thought to store them in an object like this and migrate the others in the future. WDYT?

Reply:
Rather raise it as a separate issue, I don't want to have links all over the place.
It will become messy real quick.

So either we fix all links to this format or skip it for now.

Reply:
Good point. Let's keep everything in the constants for now and I'll create a task in the backlog to clean it up.